### ISSUE TITLE
Improve first-run tutorial

### ISSUE THREAD

**Issue by BruebachL** [Cockatrice/Cockatrice#6404]
**Description of New Feature**
I would like to use this ticket to track suggestions about which parts of Cockatrice should be explained in the first-run tutorial and in which ways.

**Additional Context**
#6401 Introduced a system for a first-run tutorial within Cockatrice that allows us to highlight selected widgets and present a speech bubble next to them. Additionally, we can perform actions before and after the step is shown.

**Screenshots**
<img width="2560" height="1401" alt="image" src="https://github.com/user-attachments/assets/995be173-7072-4f8a-a8ab-e935877b9c5c" />
<img width="2560" height="1377" alt="image" src="https://github.com/user-attachments/assets/bcdba168-dd80-4d2a-b100-2d9cc8165583" />
<img width="2560" height="1375" alt="image" src="https://github.com/user-attachments/assets/c7a23690-e1f2-4ba3-b91c-f02cfc2a93c9" />
<img width="2556" height="1397" alt="image" src="https://github.com/user-attachments/assets/8523aea9-4f69-482a-89a6-e5d4d7c87444" />
<img width="2552" height="1402" alt="image" src="https://github.com/user-attachments/assets/7aaf917d-0943-457a-b823-178d5d54ea8c" />
<img width="2559" height="1400" alt="image" src="https://github.com/user-attachments/assets/0b665353-366d-4883-bb23-50cd2368b0bd" />
<img width="1723" height="527" alt="image" src="https://github.com/user-attachments/assets/cb2918bb-a9e1-4433-b222-307938bfe0f6" />
<img width="983" height="486" alt="image" src="https://github.com/user-attachments/assets/c79aa913-8656-497f-8f15-035b1de713bd" />
<img width="2087" height="593" alt="image" src="https://github.com/user-attachments/assets/3dc3b9d4-4b29-4198-b4de-dd0f7d33eb00" />
<img width="1440" height="351" alt="image" src="https://github.com/user-attachments/assets/d36674dc-7c25-442f-b7d8-a17a35ae22ab" />
<img width="941" height="348" alt="image" src="https://github.com/user-attachments/assets/93e97539-5f9c-4952-92ef-06b0edaf954a" />

**Comment by BruebachL** [comment]
<img width="2560" height="1399" alt="Image" src="https://github.com/user-attachments/assets/a33f3be5-5e94-4949-b15a-620b20f32b55" />
<img width="764" height="302" alt="Image" src="https://github.com/user-attachments/assets/ca44f5fd-7592-4f81-9d3a-fc49bb9fefdb" />
<img width="743" height="234" alt="Image" src="https://github.com/user-attachments/assets/d2f05b94-6213-4650-95ee-b5bfa837a318" />
<img width="761" height="289" alt="Image" src="https://github.com/user-attachments/assets/bdc1becd-4eab-407c-ba96-54a66b9953ca" />
<img width="753" height="277" alt="Image" src="https://github.com/user-attachments/assets/91f7e719-0212-44dc-8421-7fe1bc3acdc9" />
<img width="750" height="255" alt="Image" src="https://github.com/user-attachments/assets/9f49c6f7-3f62-4614-9401-c55dd1a23754" />
<img width="763" height="262" alt="Image" src="https://github.com/user-attachments/assets/e548116e-999d-4428-a8df-d8b35238753e" />
<img width="749" height="254" alt="Image" src="https://github.com/user-attachments/assets/bb8df259-bb14-4b14-a9e9-2a159cf59de9" />

**Comment by ebbit1q** [comment]
I think a tutorial for the home screen is superfluous, its buttons are clearly labeled, the new vde would benefit immensely from a new tutorial system, both for new players and upgrading players.
